src/org/openstreetmap/josm/plugins/conflation/command/ConflateMatchCommand.java

```diff
@@ -196,6 +198,10 @@ public static Command buildReplaceGeometryCommand(OsmPrimitive subjectObject, Os
         try {
             command = ReplaceGeometryUtils.buildReplaceCommand(subjectObject, referenceObject);
         } catch (ReplaceGeometryException ex) {
+            Collection<OsmPrimitive> problem = new HashSet<>();
+            problem.add(subjectObject);
+            problem.add(referenceObject);
+            AutoScaleAction.zoomTo(problem);
```

Comment:
can probably be simplified to `AutoScaleAction.zoomTo(Arrays.asList(subjectObject, referenceObject));`